src/journal/journalctl.c

```diff
@@ -332,6 +334,7 @@ static void help(void) {
                "  -t --identifier=STRING   Show entries with the specified syslog identifier\n"
                "  -p --priority=RANGE      Show entries with the specified priority\n"
                "  -g --grep=PATTERN        Show entries with MESSSAGE matching PATTERN\n"
+               "  --case-sensitive[=BOOL]  Force case sensitive or insenstive matching\n"
```

Comment:
Maybe instead use `-G` for case-senstivie grepping, and `-g` for non-case-sensitive grepping?

Reply:
I though about that... that can always be added. But a separate option is still needed, because there are three cases: sensitive, insensitive, automatic. So g/G is not enough.

Reply:
hmm, indenting is different from other lines...